### PR TITLE
Change README.md

### PR DESCRIPTION
**Pull request by PashaTerletskyi**

### CODE REVIEW

**Review by OlesiaKapustynska — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+* Get all cinema halls.
+* Get all movie sessions.
+* Get available movie sessions
+
+**Admin functions :**
+* Add new movies.
+* Add new cinema halls.
+* Add new movie sessions.
+* Add update movie sessions.
+* Add delete movie sessions.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* Update a movie session.
* Delete a movie session.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+
+**User functions :**
+* Get own orders.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* Get user's own orders.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,48 @@
...
+* Complete own orders in own shopping cart.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* Add a ticket to his shopping cart.
* View his shopping cart.
* Complete an order.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,48 @@
...
+## _PROJECT USES :_
+
+* Java 11.
+* Maven 4.0.0.
+* Tomcat 9.0.50.
+* MySQL Workbench 8.0.25.
+* Maven Checkstyle Plugin 3.1.1.
+* Hibernate 5.4.27.Final.
+* Spring 5.3.9.
+* Spring Security 5.5.2.
```
Comment (KseniiaMakarova) [inline README.md:43]: add Spring MVC

File: README.md [hunk 5]
```diff
@@ -1 +1,48 @@
...
+## _DATABASE STRUCTURE :_
+
+<img src="https://i.ibb.co/pWxXSww/diagram.png" alt="diagram">
+
+## _PROJECT USES :_
+
+* Java 11.
+* Maven 4.0.0.
+* Tomcat 9.0.50.
+* MySQL Workbench 8.0.25.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* MySQL
```

File: README.md [hunk 6]
```diff
@@ -1 +1,48 @@
...
+* Spring 5.3.9.
+* Spring Security 5.5.2.
+
+## _HOW TO RUN PROJECT :_
+
+1. Clone this project and open it in your IntelliJ IDEA(Add SDK(11 or above)).
+1. Install MySQL Workbench(8.0.25 or above) and configure your schema. Change your parameters in db.properties(Example below)
+
+   <img src="https://i.ibb.co/8gWcRqj/example.png" alt="example">
+1. Run the project with Tomcat configuration.
```
Comment (KseniiaMakarova) [inline README.md]: let's add the information about how we can test our functionality (we can login as admin if we need - put the login-password info here)

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1 +1,61 @@
...
+* Get all movies. Endpoint : /movies
+* Get all cinema halls. Endpoint : /cinema-halls
+* Get all movie sessions. Endpoint : /movie-sessions
+* Get available movie sessions. Endpoint : /movie-sessions/available
+
+**Admin functions :**
+* Add new movies. Endpoint : /movies
+* Add new cinema halls. Endpoint : /cinema-halls
+* Add new movie sessions. Endpoint : /movie-sessions
+* Update a movie sessions with id. Endpoint : movie-sessions/{id}
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* Update a movie sessions with id. Endpoint : /movie-sessions/{id}
```

File: README.md [hunk 8]
```diff
@@ -1 +1,61 @@
...
+## _HOW TO RUN PROJECT :_
+
+1. Clone this project and open it in your IntelliJ IDEA(Add SDK(11 or above)).
+2. Install MySQL Workbench(8.0.25 or above) and configure your schema. Change your parameters in db.properties(Example below)
+
+   <img src="https://i.ibb.co/8gWcRqj/example.png" alt="example">
+3. Run the project with Tomcat configuration. 
+4. You can test app functionality with Postman : 
+* To test admin functions use already created user (email : admin@gmail.com, password : admin)
+* To test user functions use Postman.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* To test user functions first use Postman to register a new user. 
```

File: README.md [hunk 9]
```diff
@@ -1 +1,61 @@
...
+**Admin functions :**
+* Add new movies. Endpoint : /movies
+* Add new cinema halls. Endpoint : /cinema-halls
+* Add new movie sessions. Endpoint : /movie-sessions
+* Update a movie sessions with id. Endpoint : movie-sessions/{id}
+* Delete a movie sessions with id . Endpoint : /movie-sessions/{id}
+
+**User functions :**
+* Get user's own orders. Endpoint : /orders
+* Add a ticket with movie session to his shopping cart. Endpoint : /movie-sessions?userId&movieSessionId
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* Add a ticket with movie session to his shopping cart. Endpoint : /shopping-carts/movie-sessions
```

File: README.md [hunk 10]
```diff
@@ -1 +1,61 @@
...
+* View his shopping cart. Endpoint : /shopping-carts/by-user?userId
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* View his shopping cart. Endpoint : /shopping-carts/by-user
```

File: README.md [hunk 11]
```diff
@@ -1 +1,61 @@
...
+* Complete an order. Endpoint : /complete
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
* Complete an order. Endpoint : /orders/complete
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — COMMENTED** [review]
overall looks great) 👍

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**